pages/_error.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import { FsaLayout } from "../src/components";
+import { Header, Paragraph, BackLink, Button } from "govuk-react";
+
+class Error extends React.Component {
+  static getInitialProps({ req, res, err }) {
+    let statusCode;
+    if (res) {
+      statusCode = res.statusCode;
+    } else if (err) {
+      statusCode = err.statusCode;
+    } else {
+      statusCode = null;
+    }
+    const council = req.session.council;
+    const referrer = req.header("Referrer") || "/";
+    return { statusCode, council, referrer };
+  }
+
+  render() {
+    const backToStartLink =
+      this.props.council === undefined ? `/` : `/new/${this.props.council}`;
+    switch (parseInt(this.props.statusCode, 10)) {
+      case 404:
+        return (
+          <FsaLayout {...this.props}>
+            <BackLink href={backToStartLink}>Back to start</BackLink>
+            <Header level={1}>Page Not Found</Header>
+            <Paragraph>
+              Please contact your Local Council if you need to speak to someone
+              about your food business registration urgently.
+            </Paragraph>
+            <form id="unknown-error-form" action={this.props.referrer}>
```

Comment:
If the referrer is unknown, you can't take the user to the previous page, so wouldn't it make sense to hide this button in this case?